src/keys-builder/typescript/build-keys-from-ast-nodes.ts

```diff
@@ -30,22 +31,32 @@ export function buildKeysFromASTNodes(
       const data = {};
 
       const [key, _, lang] = node.parent.arguments;
```

Comment:
I think we need to make some changes due to your addition.
```ts
const [keyNode, _, langNode] = node.parent.arguments;
let lang: string;
let keys: string[];

if (isStringNode(langNode)) {
  lang = langNode.text;
}

if (isStringNode(keyNode)) {
  keys = [keyNode.text]
} else if (isArrayLiteralExpression(keyNode)) {
  keys = keyNode.elements.filter(isStringNode).map((node) => node.text);
}

for(const key of keys) {
  const data = lang ? {lang, key} : {key};
  result.push(data);
}
```

WDYT? I think it's clearer.

Reply:
I agree, I was tempted to fix it earlier, but I didn't want to refactor too much on my first PR.